citest/gcp_testing/gce_util.py

```diff
@@ -287,7 +287,7 @@ def establish_network_connectivity(gcloud, instance, target_port):
     atexit.register(_ProcessKiller(pid).safe_kill)
 
   logger.debug('Confirming tunnel is working')
-  url = 'http://localhost:%d' % local_port
+  url = 'http://localhost:%d/health' % local_port
```

Comment:
This isnt right. This module is in citest, not spinnaker, so is intended to be generic. To add this, you need to add a "basepath" attribute and have spinnaker add the /health path.

Since this is already merged, I'll look into fixing it.